rust/stackable-cockpit/src/platform/demo/spec.rs

```diff
@@ -80,31 +124,30 @@ impl DemoSpecV2 {
         transfer_client: &FileTransferClient,
         skip_release: bool,
     ) -> Result<(), DemoError> {
-        // Returns an error if the demo doesn't support to be installed in the
-        // requested namespace
-        if !self.supports_namespace(product_namespace) {
-            return Err(DemoError::UnsupportedNamespace {
-                requested: product_namespace.to_string(),
-                supported: self.supported_namespaces.clone(),
-            });
-        }
-
         // Get the stack spec based on the name defined in the demo spec
         let stack_spec = stack_list.get(&self.stack).ok_or(DemoError::NoSuchStack {
             name: self.stack.clone(),
         })?;
 
-        // Install the stack
+        // Check prerequisites
         stack_spec
-            .install(
+            .check_prerequisites(product_namespace)
+            .await
+            .context(StackSnafu)?;
```

Comment:
Any particular reason why this lives here instead of the `stack::install_release` function? This would make sure we also check the prerequisites when *only* installing the stack.

Reply:
Two things:
1. IMHO the sanity checks regarding the **stack** do not belong in a fn `stack::install_release`
2. Before this change when installing a stack we actually got the same warnings twice

I know that it's replicated two times, but I think these are the only places we will need to check this looking into the future using my magic fortune telling ball.

Reply:
Okay I will trust your magic fortune telling ball and we can leave it as is. If we ever need it to change, we can do so in the future :)